include/tables/scene_table.h

```diff
@@ -9,114 +9,114 @@
+/* 0x16 */ DEFINE_SCENE(MIZUsin_bs_scene, none, SCENE_WATER_TEMPLE_BOSS, SDC_WATER_TEMPLE_BOSS, 0, 0)
+/* 0x17 */ DEFINE_SCENE(jyasinboss_scene, none, SCENE_SPIRIT_TEMPLE_BOSS, SDC_DEFAULT, 0, 0)
+/* 0x18 */ DEFINE_SCENE(HAKAdan_bs_scene, none, SCENE_SHADOW_TEMPLE_BOSS, SDC_SHADOW_TEMPLE, 0, 0)
+/* 0x19 */ DEFINE_SCENE(ganon_boss_scene, none, SCENE_GANONDORF_BOSS, SDC_DEFAULT, 0, 0)
+/* 0x1A */ DEFINE_SCENE(ganon_final_scene, none, SCENE_GANONS_CASTLE_COLLAPSE, SDC_GANONS_CASTLE_COLLAPSE, 0, 0)
+/* 0x1B */ DEFINE_SCENE(entra_scene, none, SCENE_MARKET_ENTRANCE_DAY, SDC_DEFAULT, 0, 0)
+/* 0x1C */ DEFINE_SCENE(entra_n_scene, none, SCENE_MARKET_ENTRANCE_NIGHT, SDC_DEFAULT, 0, 0)
+/* 0x1D */ DEFINE_SCENE(enrui_scene, none, SCENE_MARKET_ENTRANCE_RUINS, SDC_DEFAULT, 0, 0)
+/* 0x1E */ DEFINE_SCENE(market_alley_scene, g_pn_18, SCENE_BACK_ALLEY_DAY, SDC_DEFAULT, 0, 0)
+/* 0x1F */ DEFINE_SCENE(market_alley_n_scene, g_pn_18, SCENE_BACK_ALLEY_NIGHT, SDC_DEFAULT, 0, 0)
 /* 0x20 */ DEFINE_SCENE(market_day_scene, g_pn_17, SCENE_MARKET_DAY, SDC_DEFAULT, 0, 0)
 /* 0x21 */ DEFINE_SCENE(market_night_scene, g_pn_17, SCENE_MARKET_NIGHT, SDC_DEFAULT, 0, 0)
 /* 0x22 */ DEFINE_SCENE(market_ruins_scene, g_pn_17, SCENE_MARKET_RUINS, SDC_DEFAULT, 0, 0)
-/* 0x23 */ DEFINE_SCENE(shrine_scene, none, SCENE_SHRINE, SDC_DEFAULT, 0, 0)
-/* 0x24 */ DEFINE_SCENE(shrine_n_scene, none, SCENE_SHRINE_N, SDC_DEFAULT, 0, 0)
-/* 0x25 */ DEFINE_SCENE(shrine_r_scene, none, SCENE_SHRINE_R, SDC_DEFAULT, 0, 0)
-/* 0x26 */ DEFINE_SCENE(kokiri_home_scene, none, SCENE_KOKIRI_HOME, SDC_DEFAULT, 0, 0)
-/* 0x27 */ DEFINE_SCENE(kokiri_home3_scene, none, SCENE_KOKIRI_HOME3, SDC_DEFAULT, 0, 0)
-/* 0x28 */ DEFINE_SCENE(kokiri_home4_scene, none, SCENE_KOKIRI_HOME4, SDC_DEFAULT, 0, 0)
-/* 0x29 */ DEFINE_SCENE(kokiri_home5_scene, none, SCENE_KOKIRI_HOME5, SDC_DEFAULT, 0, 0)
-/* 0x2A */ DEFINE_SCENE(kakariko_scene, none, SCENE_KAKARIKO, SDC_DEFAULT, 0, 0)
-/* 0x2B */ DEFINE_SCENE(kakariko3_scene, none, SCENE_KAKARIKO3, SDC_DEFAULT, 0, 0)
-/* 0x2C */ DEFINE_SCENE(shop1_scene, g_pn_23, SCENE_SHOP1, SDC_DEFAULT, 0, 0)
+/* 0x23 */ DEFINE_SCENE(shrine_scene, none, SCENE_TEMPLE_OF_TIME_EXTERIOR_DAY, SDC_DEFAULT, 0, 0)
+/* 0x24 */ DEFINE_SCENE(shrine_n_scene, none, SCENE_TEMPLE_OF_TIME_EXTERIOR_NIGHT, SDC_DEFAULT, 0, 0)
+/* 0x25 */ DEFINE_SCENE(shrine_r_scene, none, SCENE_TEMPLE_OF_TIME_EXTERIOR_RUINS, SDC_DEFAULT, 0, 0)
+/* 0x26 */ DEFINE_SCENE(kokiri_home_scene, none, SCENE_KNOW_IT_ALL_BROS_HOUSE, SDC_DEFAULT, 0, 0)
+/* 0x27 */ DEFINE_SCENE(kokiri_home3_scene, none, SCENE_TWINS_HOUSE, SDC_DEFAULT, 0, 0)
+/* 0x28 */ DEFINE_SCENE(kokiri_home4_scene, none, SCENE_MIDOS_HOUSE, SDC_DEFAULT, 0, 0)
+/* 0x29 */ DEFINE_SCENE(kokiri_home5_scene, none, SCENE_SARIAS_HOUSE, SDC_DEFAULT, 0, 0)
+/* 0x2A */ DEFINE_SCENE(kakariko_scene, none, SCENE_KAKARIKO_CENTER_HOUSE, SDC_DEFAULT, 0, 0)
+/* 0x2B */ DEFINE_SCENE(kakariko3_scene, none, SCENE_BACK_ALLEY_HOUSE, SDC_DEFAULT, 0, 0)
```

Comment:
after giving all of these names one last look through, `SCENE_KAKARIKO_CENTER_HOUSE` is the only one I think could use some work
I had to ask myself what the house in the center of kakariko was, and I still guessed wrong. I thought it was house of skulltula until I saw its define further down lol

do understand this one is pretty hard to name tho. I think it would be better off named after an "inn" somehow, but idk

Reply:
it may not be explicitly called an Inn in-game anywhere, but it houses multiple different people, and is the only house in the area that *could* be considered an inn. while I think what is considered the center of kakariko is a bit subjective (mainly because of its awkward shape)

but if there are any other suggestions than inn that could be better someone feel free to voice them

Reply:
Copypasting from previous discussions/summaries:

<details>

### `kakariko_scene`

(the house in the middle of kakariko that houses carpenters at night as child, and Talon as adult before he goes back to lonlon)

Encyclopedia says this is a "Shelter" as adult ![](https://cdn.discordapp.com/attachments/1014952883709562960/1014961188242735134/unknown.png)

- `KAKARIKO_GUEST_HOUSE`
- `KAKARIKO_CENTER_HOUSE`

</details>

<details>

https://discord.com/channels/688807550715560050/1014952883709562960/1014956822949540011

> maybe naming houses by their appearance / location when naming by function is unclear? though I don't actually have any suggestions for the carpenter house. GUEST_HOUSE and CENTER_HOUSE both seemed fine to me. this is hard

kentonm

> I agree with location, CENTER_HOUSE makes the most sense to me in this situation

caverill

https://discord.com/channels/688807550715560050/1014952883709562960/1029046975728996422

> for the central kak house I think it's nice to have the location in the name since most people know where it's located in kakariko, I'm okay with kakariko_center_house

yanis

</details>

I myself don't have an opinion, I picked the one that had more positive feedback

Reply:
`KAKARIKO_GUEST_HOUSE` is fine with me as well if not inn. but yeah I still dont agree with location-based, cause I prob wont be the only one to be confused about what the center actually is